curated_transformers/models/feedforward.py

```diff
@@ -25,15 +25,15 @@ class PointwiseFeedForward(Module):
 
     This layer is applied pointwise, meaning that the same
     transformation is applied to each sequence element. This
-    transformation is: g(xW_1 + b_1)W_2 + b_2, where W_1 and b_1 transform
-    the input to an intermediate width, g is a non-linear activation
-    function and W_2 and b_2 transform the output of the activation back to
+    transformation is: ``g(xW_1 + b_1)W_2 + b_2``, where ``W_1`` and ``b_1``
```

Comment:
Maybe we want to convert these paragraphs to proper LaTeX math as well, since it's already pseudo-LaTeX?

Reply:
Converted.